.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -8,8 +8,20 @@ jobs:
     vmImage: ubuntu-latest
   strategy:
     matrix:
-      linux_64_:
-        CONFIG: linux_64_
+      linux_64_cuda_compiler_version11.0cxx_compiler_version9fortran_compiler_version9:
```

Comment:
Is the plain Linux (no cuda, no aarch64) build missing? I don't see it here or in checks, but I'm on a restricted screen.

Reply:
The plain Linux build now goes under the name `linux_64_cuda_compiler_versionNone*`, similar to the Windows build which is now named `win_64_cuda_compiler_versionNone`.